lib/mariaex/geometry.ex

```diff
@@ -0,0 +1,23 @@
+if Code.ensure_loaded?(Ecto.Type) do
```

Comment:
This doesn't belong here! `Ecto.Type` wont be loaded when most ppl build. Also no `Ecto` stuff please because that creates circular dependency.